API/Model/Product.cs

```diff
@@ -7,5 +7,6 @@ public class Product
         public string? Description { get; set; }
         public decimal? Price { get; set; }
         public int? AmountInStock { get; set; }
+        public List<OrderItem> OrderItems { get; set; }
```

Comment:
nemanau kad produktui reikia turet sito

Reply:
Tada, kaip aprašyti Product Entity context'e? Ar nereikia?

Reply:
not sure, manau sito fieldo nereik kad aprasyt

Reply:
Tada gali Product pažiūrėt contexte? Nesugalvoju, kaip kitaip

Reply:
Jei ištrini OrderItems, tai braukia context faile jį prie Product